### PR TITLE
Add `yasm` & `openslide` for cuCIM wheel builds

### PR DESCRIPTION
**Pull request by jakirkham**
Per request: https://github.com/rapidsai/cucim/pull/619#discussion_r1382128395

cc @raydouglass @gigony @grlee77

### CODE REVIEW

**Review by raydouglass — APPROVED** [review]

File: ci-wheel.Dockerfile [hunk 1]
```diff
@@ -34,7 +34,7 @@ RUN case "${LINUX_VER}" in \
           jq wget gcc zlib1g-dev libbz2-dev \
           libssl-dev libreadline-dev libsqlite3-dev libffi-dev curl git libncurses5-dev \
           libnuma-dev openssh-client libcudnn8-dev zip libopenblas-dev liblapack-dev \
-          protobuf-compiler autoconf automake libtool cmake \
+          protobuf-compiler autoconf automake libtool cmake yasm openslide-dev \
```
Comment (grlee77) [inline ci-wheel.Dockerfile:37]:
```suggestion
          protobuf-compiler autoconf automake libtool cmake yasm libopenslide-dev \
```

File: ci-wheel.Dockerfile [hunk 2]
```diff
@@ -71,14 +74,17 @@ RUN case "${LINUX_VER}" in \
...
-          epel-release which wget gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite \
+          which wget gcc zlib-devel bzip2 bzip2-devel readline-devel sqlite \
           sqlite-devel xz xz-devel libffi-devel curl git ncurses-devel numactl \
-          numactl-devel openssh-clients libcudnn8-devel zip jq \
+          numactl-devel openssh-clients libcudnn8-devel zip jq openslide-devel \
           protobuf-compiler autoconf automake libtool dnf-plugins-core cmake \
         && dnf config-manager --set-enabled powertools \
         && dnf install -y blas-devel lapack-devel \
         && dnf -y install gcc-toolset-11-gcc gcc-toolset-11-gcc-c++ \
+        && dnf -y yasm \
```
Comment (grlee77) [inline ci-wheel.Dockerfile:87]:
```suggestion
        && dnf install -y yasm \
```

Comment (jakirkham) [reply ci-wheel.Dockerfile]: Done in commit: b77c613eb92dc42d0600817d504b2797cbfbba8d

**Review by grlee77 — APPROVED** [review]
Thanks, LGTM

**Review by raydouglass — APPROVED** [review]

**Pull request merged**